src/main/kotlin/org/rust/lang/refactoring/RustRefactoringSupportProvier.kt

```diff
@@ -2,10 +2,21 @@ package org.rust.lang.refactoring
 
 import com.intellij.lang.refactoring.RefactoringSupportProvider
 import com.intellij.psi.PsiElement
+import com.intellij.refactoring.RefactoringActionHandler
 import org.rust.lang.core.psi.RustPatBindingElement
 
 class RustRefactoringSupportProvider : RefactoringSupportProvider() {
     override fun isInplaceRenameAvailable(element: PsiElement, context: PsiElement?): Boolean {
         return element is RustPatBindingElement
     }
+
+    override fun getIntroduceVariableHandler(): RefactoringActionHandler? {
```

Comment:
You can write ` override fun getIntroduceVariableHandler() = RustLocalVariableHandler()`.